LayoutTests/fast/images/decode-static-svg-image-resolve.html

```diff
@@ -0,0 +1,38 @@
+<style>
+/* This tests passes if two green rectangles are drawn. */
```

Comment:
Please remove this comment. It is already in the body as a `<p>` element.

Reply:
Done.